src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -0,0 +1,52 @@
+import {ChatInputCommandInteraction, inlineCode, PermissionFlagsBits} from "discord.js";
+import {SlashCommandBuilder, SlashCommandScope} from "../../../builders/SlashCommandBuilder";
+import {useChatCommand} from "../../../hooks/useChatCommand";
+import axios from 'axios';
+
+const builder = new SlashCommandBuilder()
+    .setName("jumbo")
+    .setDescription("Make G'day blow up an image.")
+    .addStringOption(option => option.setName("emoji").setDescription("Emoji to jumbo size.").setRequired(true))
+    .setDefaultMemberPermissions(PermissionFlagsBits.ManageMessages)
+    .setEphemeral(true)
+    .setScope(SlashCommandScope.GLOBAL);
```

Comment:
`SlashCommandScope.MAIN_GUILD` is preferred as g'day is used in other servers for g'bye that likely don't want the clutter of extra (likely conflicting) commands

Reply:
I can switch it to MAIN_GUILD, I think I interpreted GLOBAL as applicable to both MAIN and STAFF, which I wanted to assume is the preference.